share/extensions/MSXPi.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" ?>
+<!DOCTYPE msxconfig SYSTEM 'msxconfig2.dtd'>
+<msxconfig>
+  <info>
+    <name>MSXPi</name>
+    <manufacturer>MSXPi (https://github.com/costarc/MSXPi)</manufacturer>
+    <code/>
+    <release_year/>
+    <description>MSXPi is an extension for openMSX that emulates the MSXPi hardware interface, enabling an MSX computer to interact with modern systems. Unlike the physical version, the emulated interface does not require a Raspberry Pi — it communicates with a Python-based server component that provides access to web resources, files, REST APIs, and other external services.</description>
+    <type>pluggable</type>
+  </info>
+  <devices>
+    <primary slot="any">
+      <secondary slot="any">
+        <MSXPiDevice id="MSXPi">
+          <io base="0x56" num="8"/>

```

Comment:
0x56..0x5a that's 5 ports, right? Not 8.
But even better would be to really only register the required ports:
```
<io base="0x56" num="1"/>
<io base="0x5a" num="1"/>
```
Unless you expect that future versions will use more ports than just 0x56 and 0x5a? You know the hardware and your future plans better, you decide.

Reply:
That (8 ports) reflects the current state of the physical hardware documentation, as originally designed. Does not make difference if I change the extension to 5, since the ports 0x5B-0x5D are not used anyway. But same is true to ports 0x57-0x59, reason I originally accepted you sugestion to change ports assignment and use only 0x56 and 0x57.

I now wnt to keep the design (both extension and hardware) as is, but then I think would be better to have the MSXPi.xml configured to use only two ports:
<io>
  <port>0x56</port>
  <port>0x5A</port>
</io>
I suppose that syntax is accepted by openMSX?

Is this gap going to be a problem other than just inconvenience ? If there are no objections, I would proceed with this change.

Reply:
> 0x56..0x5a that's 5 ports, right? Not 8. But even better would be to really only register the required ports:
> 
> ```
> <io base="0x56" num="1"/>
> <io base="0x5a" num="1"/>
> ```
> 
> Unless you expect that future versions will use more ports than just 0x56 and 0x5a? You know the hardware and your future plans better, you decide.

That (8 ports) reflects the current state of the physical hardware documentation, as originally designed. Does not make difference if I change the extension to 5, since the ports 0x5B-0x5D are not used anyway. But same is true to ports 0x57-0x59, reason I originally accepted you sugestion to change ports assignment and use only 0x56 and 0x57.

> <io base="0x56" num="1"/>
> <io base="0x5a" num="1"/>

Yes, this is the best option.